src/input-range-select/input-range-select.tsx

```diff
@@ -134,6 +136,13 @@ export const InputRangeSelect = <T, V>({
         }
     };
 
+    const handleClear = (event: React.MouseEvent) => {
+        event.stopPropagation();
+        setSelectedFromValue(undefined);
+        setSelectedToValue(undefined);
+        onSelectOption(undefined, undefined);
```

Comment:
based on the types the first param should be an object containing the from/to properties

`onSelectOption` is also optional so it's best to check if it is defined before calling it

```suggestion
        onSelectOption({ from: undefined, to: undefined }, undefined);
```